### PR TITLE
refactor(logging): centralizes logging and mqtt messages into messages.ts

### PR DESCRIPTION
**Pull request by matt-primrose**
## PR Checklist

- [ ] Unit Tests have been added for new changes
- [ ] API tests have been updated if applicable
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with Apache 2.0 and clearly outlined the added dependency.

## What are you changing?
moves all logging and mqtt messages into a single messages.ts file under a new logging folder
moves logger.ts into logging folder
fixes some log messages
unifies structure of common messages

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
 * SPDX-License-Identifier: Apache-2.0
 **********************************************************************/
 
-import { logger as log } from './utils/logger'
+import { logger as log } from './logging/logger'
```
Comment (rsdmike) [inline src/index.ts:6]: update references in the fille to use logger instead of log

**Review by rsdmike — CHANGES_REQUESTED** [review]
we've got a bit more stuttering with the naming here than ideal. I think if you rename the logger file to index.ts that would help a bit. Additionally, logger.silly(silly.message) also stutters. Not sure about the best way to handle that at the moment.

File: src/index.ts (hunk 1 above)
Comment (matt-primrose) [reply src/index.ts]: Updated

**Review by rsdmike — APPROVED** [review]

**Comment by rsdmike** [comment]
Messages should be all caps for continuity

**Pull request merged**